org-brain.el

```diff
@@ -1256,11 +1291,11 @@ If ENTRY isn't specified, ask for the ENTRY."
   (org-brain-goto entry #'pop-to-buffer))
 
 ;;;###autoload
-(defun org-brain-goto-end (&optional entry)
+(defun org-brain-goto-end (&optional entry same-window)
```

Comment:
Maybe it should now be `(interactive "P")` and work as the other `goto-x` commands? Also the docstring should be updated.

Reply:
Hmm I'm confused. What do you mean by "work as the other `goto-x` commands? The goto-x functions don't have consistent usage — some use `(interactive "P")`, some use `(interactive (list (org-brain-entry-at-pt)))`, some use `(interactive)`. And in my view org-brain-goto-end should act very similar to org-brain-goto, since the main difference is in where the cursor winds up after the entry has been jumped to. Could you expand on what consistency you would like to see here?

Reply:
You're right, for some reason I thought all of them had the same arguments. My bad! Your suggestion works.